package.json

```diff
@@ -1,19 +1,17 @@
 {
   "name": "fitext",
-  "version": "1.0.5.c",
+  "version": "1.0.5",
   "description": "Resizes text elements proportionally to fit any element",
-  "main": "dist/fitext.cjs.js",
-  "module": "dist/fitext.esm.js",
-  "browser": "dist/fitext.umd.js",
+  "main": "dist/fitext.umd.js",
   "source": "src/fitext.js",
   "types": "src/types.d.ts",
   "scripts": {
     "dev": "npx webpack-dev-server",
-    "build": "klap build"
+    "build": "npx klap build --no-sourcemap -b --browserslist"
   },
   "browserslist": [
```

Comment:
Hi @osdevisnot,

Nice plugin ! I've updated some of your changes in the PR. I've checked the babel processing of the code. I've an issue on it : klap uses all the time his own browserslist properties and not these I've defined in the package.json 🤔.

By the way I don't understand very well what implicate the modules (ESM, UMD, CJS). I just want to get IE compatibility and deferable loading and possibility to import it as node_module indeed. Can you advice on it ?

Reply:
`klap` should read `browserslist` property from `package.json` - but it seems it was looking for `browserlist` - notice the mussing `s` from spelling 😢  I've published a new version of `klap` to fix this.

and here is a short summary on `umd` vs `esm` vs `cjs`:
- you should publish your package in `cjs` format. nodejs looks at `main` property in your package.json to determine which file to load.
- you want to publish your package in `esm` format. so that you can import your code directly in modern browsers and tools like rollup/webpack/parcel can use this format for better treeshaking when you library is used in application
- you want to publich your package in `umd` as a fallback for users wanting to use your package directly with script tag in IE11 (older browsers)

It is therefore, generally advised to create all three formats for best results in all the target environments.

klap automates this for you, by looking at `main`, `module` and `browser` properties in your `package.json`

Reply:
In the last commit I've changed the browserlist behavior to cover the last 2 chrome versions (to check if browserlists works with my settings). But nothing changed about the size and the content, the browserslist flag never get my settings to process the file.

Reply:
I think I might need to work on documentation for flags. 

When you want to specify browserslist, you either use 
- `klap build -b 'target'` or 
- `klap build --browserslist 'target'`

Passing it like `-b --browserslist` causes `--browserslit` to the value for browserlist.

which is why your `browserslist` from `package.json` was never picked up.